tests/functional/utils/test_misc.py

```diff
@@ -169,3 +173,15 @@ def my_method():
 
     my_method()
     assert "Oh no!" in ape_caplog.head
+
+
+def test_get_current_timestamp_ms():
+    """
+    Proving the current timestamp is UTC.
+    """
+    actual = get_current_timestamp_ms()  # UTC
+    zone = ZoneInfo("America/Chicago")
+    dt = datetime(2020, 10, 31, 12, tzinfo=zone)
+    unexpected = round(datetime.now(tz=dt.tzinfo).timestamp() * 1000)
+    diff = actual - unexpected
+    assert diff == pytest.approx(18_000_000)  # Difference of CST to UTC.
```

Comment:
```suggestion
    assert diff == pytest.approx(18_000_000) or diff == pytest.approx(19_000_000) # Difference of CST to UTC.
```

to account for daylight savings i guess